source/triggers/clauses/clauseProto_OnEvent.js

```diff
@@ -71,7 +71,7 @@ this.reset = function() {
 
 this.evaluateClause = function() {
     var retVal = ( this.lastEventTime$ > 0 ) && 
-                 ( Date.now() - this.lastEventTime$ <= this.threshold$ );
+                 ( this.time - this.lastEventTime$ <= this.threshold$ );
```

Comment:
I thought that we needed to use Date.now() (a) to ensure that this replicates, and (b) to ensure that we don't have framerate issues.  If you use this.time and the framerate is very low, you might get outside of the threshold in which the event is considered "current" before it ever has a chance to fire.  I would not switch this back.